automation/vars/default.yaml

```diff
@@ -271,8 +271,10 @@ vas:
         post_stage_run:
           - name: Deploy Ceph
             type: playbook
-            source: "../../playbooks/ceph.yml"
+            source: "../../hooks/playbooks/ceph-bm.yml"
```

Comment:
I don't see this listed here: https://github.com/openstack-k8s-operators/ci-framework/tree/main/hooks/playbooks

Reply:
Hi Andrew it is in this patch https://github.com/openstack-k8s-operators/ci-framework/pull/2246/files that we are working on getting merged